marxs/source/labSource.py

```diff
@@ -105,3 +108,74 @@ def generate_photons(self, exposuretime):
         photons.add_column(Column(name='dir', data=dir.T))
         photons.add_column(Column(name='polarization', data=polarization_vectors(dir.T, photons['polangle'])))
         return photons
+
+class LabPointSourceCone(Source):
+    '''In-lab point source
+
+    - photons uniformly distributed in all directions in cone
+    - photon start position is source position
```

Comment:
Text is better than bullet point for documentation.
Start with one line summary, one empty line, then text like this:
```
Lab point source that emits uniformly into a cone.

Photons all emerge from a point source. Unlike the other laboratory sources, only a defined cone is illuminated.
```